gc_project/gc_project/urls.py

```diff
@@ -23,7 +24,7 @@
                          LoginWidgetView, UserView)
 from gears.views import (HomeView, CategoriesView, CategoryByNameView,
                          GearView, AddGearView, LocationsView,
-                         LocationByNameView)
+                         LocationByNameView, CreateCodeView, ValidateCodeView)
```

Comment:
I feel our urls.py is getting ugly with all these imports of individuals views. What you think if we "import gears.views" and just change the urlpatterns list to use views "gear.views.ViewName()"??